src/reducers/articleReducer.js

```diff
@@ -0,0 +1,22 @@
+import { POST_ARTICLE } from "../actions/types";
+import { alert } from "../actions/loginActions";
```

Comment:
@joshNic Good work👍🏾, it's good that you are reusing already existing functionality instead of making duplicates.